run.ps1

```diff
@@ -0,0 +1,2 @@
+echo "Booting docker-compose production file"
```

Comment:
Acho melhor deixar esses scripts em uma pasta separada. Uma pasta `scripts` talvez

Reply:
Consider it done as well!